src/utils/accounts.js

```diff
@@ -57,3 +58,31 @@ export function getActiveAccountWalletId(accounts: Accounts = []): string {
 export function checkIfSmartWalletAccount(account: Account): boolean {
   return account.type === ACCOUNT_TYPES.SMART_WALLET;
 }
+
+export function getAccountName(accountType: AccountTypes): string {
+  switch (accountType) {
+    case ACCOUNT_TYPES.SMART_WALLET:
+      return 'Smart Wallet';
+    case ACCOUNT_TYPES.KEY_BASED:
+      return 'Key Based account';
+    default:
+      return '';
+  }
+}
+
+export function getInactiveUserAccounts(accounts: Accounts): UserAccount[] {
```

Comment:
This method is so confusing. What is the difference between Account and UserAccount? We need to name the second better